tests/backend/test_http.py

```diff
@@ -0,0 +1,217 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2016-2021 Scille SAS
+
+import pytest
+
+from parsec import __version__ as parsec_version
+from parsec.api.protocol import OrganizationID, InvitationToken, InvitationType
+from parsec.core.types.backend_address import BackendInvitationAddr
+from parsec.backend.asgi import MAX_CONTENT_LENGTH, app_factory
+
+from tests.common import customize_fixtures
+
+
+async def _do_test_redirect(client):
+    # No redirection header. shouln't redirect.
+    rep = await client.get("/test")
+    assert rep.status == "404 NOT FOUND"
+
+    # Incorrect redirection header with good redirection protocol. shouln't redirect.
+    rep = await client.get("/test", headers={"X-Forwa-P": "https"})
```

Comment:
line 19, the header is incorrect because it is `X-Forwa-P` where it is expected to be `X-Forwa-Proto` according to `backend_forward_proto_enforce_https` fixture customization